### ISSUE TITLE
Issue migrating from MediaPlayer to FFmpegMediaPlayer, streaming with radios

### ISSUE THREAD

**Issue by 4gus71n** [wseemann/FFmpegMediaPlayer#51]
Hi! and first of all thanks for this awesome library It really boost our app performance! We developed a radio streaming app, the migration went really smooth no problems at all. But we find that when we try to play several radios at the same time the app crashes. Here is the log message:

```
1675-1905/com.arzion.streame A/libc﹕ Fatal signal 11 (SIGSEGV) at 0x00000000 (code=1), thread 1905 (AsyncTask #2)
05-27 12:07:15.063      859-942/? W/InputDispatcher﹕ channel '429703b0 com.arzion.streame/com.arzion.streame.screens.main.MainActivity (server)' ~ Consumer closed input channel or an error occurred.  events=0x9
05-27 12:07:15.063      859-942/? E/InputDispatcher﹕ channel '429703b0 com.arzion.streame/com.arzion.streame.screens.main.MainActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
05-27 12:07:15.064      859-942/? W/InputDispatcher﹕ channel '429705f0 com.arzion.streame/com.arzion.streame.screens.categorylisting.CategoryListingActivity (server)' ~ Consumer closed input channel or an error occurred.  events=0xd
05-27 12:07:15.064      859-942/? E/InputDispatcher﹕ channel '429705f0 com.arzion.streame/com.arzion.streame.screens.categorylisting.CategoryListingActivity (server)' ~ Channel is unrecoverably broken and will be disposed!
05-27 12:07:15.180      859-991/? W/InputDispatcher﹕ Attempted to unregister already unregistered input channel '429703b0 com.arzion.streame/com.arzion.streame.screens.main.MainActivity (server)'
05-27 12:07:15.192      859-870/? W/InputDispatcher﹕ Attempted to unregister already unregistered input channel '429705f0 com.arzion.streame/com.arzion.streame.screens.categorylisting.CategoryListingActivity (server)'
05-27 12:07:15.194     859-1126/? W/ActivityManager﹕ Scheduling restart of crashed service com.arzion.streame/.services.MYService in 1000ms
05-27 12:07:15.196     859-1126/? W/ActivityManager﹕ Force removing ActivityRecord{4294db08 u0 com.arzion.streame/.screens.categorylisting.CategoryListingActivity t659}: app died, no saved state
05-27 12:07:15.262    1212-3145/? W/ContextImpl﹕ Calling a method in the system process without a qualified user: android.app.ContextImpl.sendBroadcast:1252 android.content.ContextWrapper.sendBroadcast:365 com.motorola.motocare.util.TriggerHelper$TriggerBuilder.send:76 com.motorola.motocare.internal.frameworkevents.PauseResumeTrigger.handleFrameworkEvent:53 com.motorola.motocare.internal.frameworkevents.FwEventMonitor$FrameworkListener.processFrameworkEvent:114
05-27 12:07:15.359    1928-1928/com.arzion.streame W/dalvikvm﹕ VFY: unable to resolve virtual method 607: Landroid/content/res/TypedArray;.getChangingConfigurations ()I
05-27 12:07:15.363    1928-1928/com.arzion.streame W/dalvikvm﹕ VFY: unable to resolve virtual method 629: Landroid/content/res/TypedArray;.getType (I)I
05-27 12:07:15.471    1928-1928/com.arzion.streame W/dalvikvm﹕ VFY: unable to resolve virtual method 449: Landroid/content/pm/PackageManager;.getPackageInstaller ()Landroid/content/pm/PackageInstaller;
05-27 12:07:15.515    1928-1928/com.arzion.streame E/AudioManagerAndroid﹕ BLUETOOTH permission is missing!
05-27 12:07:15.644    1928-1928/com.arzion.streame W/linker﹕ libavcodec.so has text relocations. This is wasting memory and is a security risk. Please fix.
05-27 12:07:16.888    1928-1928/com.arzion.streame W/dalvikvm﹕ VFY: unable to resolve virtual method 3060: Landroid/webkit/WebSettings;.setMixedContentMode (I)V
05-27 12:07:16.913    1928-1960/com.arzion.streame W/chromium﹕ [WARNING:proxy_service.cc(890)] PAC support disabled because there is no system implementation
05-27 12:07:17.477    1928-2017/com.arzion.streame W/ActivityThread﹕ ClassLoader.loadClass: The class loader returned by Thread.getContextClassLoader() may fail for processes that host multiple applications. You should explicitly specify a context class loader. For example: Thread.setContextClassLoader(getClass().getClassLoader());
```

There's no specific message error from any call of the FFmpegMediaPlayer API methods, but that's the only thing that we've changed. This message get our attention "No account for auth token provided" can be that the radio streams that requires authentication could throw RuntimeException if no authentication has been  set at all?. Thanks!

**Comment by pfaffenrodt** [comment]
https://developer.android.com/ndk/guides/ndk-stack.html

**Comment by astinx** [comment]
Hi, thanks for reply. I'm going to log the stack right now, BTW the issue appears to be the exact same one that this https://github.com/wseemann/FFmpegMediaPlayer/issues/29 Becuase the way to reproduce the bug is exactly the same

**Comment by wseemann** [comment]
This should be fixed in version [1.0.1](https://github.com/wseemann/FFmpegMediaPlayer/releases/tag/v1.0.1).
